lib/addresses.ts

```diff
@@ -55,6 +60,7 @@ export const addresses = {
 			registry: '0xe2C16936413D74c667aeF0D040c920BF639067d4',
 			sTokens: '0xe0af15141ABd0B31Fb15e250971936Fe8837230a',
 			withdraw: '0x5f9d9f40f4d7d8A350745D69e94E20339Df17b83',
+			swap: '',
```

Comment:
`addresses.ts` is the address map for core modules, so how about declaring a new address map in `lib/agent/common/const.ts`?

Also, can you write the addresses of deployed contracts instead of empty addresses?